HULKs/hulk
# Visual referee message transmission decision

## Introduced Changes
Introduces a queue to reduce the number of messages sent in `Initial` and reduce false positives.
At the moment the queue has a length of 4 and if there are 3 detections out of those 4 the robot changes to `Ready`. Please suggest other values if this is not sufficient.
A detected pose can be seen by two purple leds in the left eye of the NAO. 

Depends on #806 

Addresses #818

## ToDo / Known Issues

*If this is a WIP describe which problems are to be fixed.*

## Ideas for Next Iterations (Not This PR)

*If there are some improvements that could be done in a next iteration, describe them here.*

## How to Test
Test with NAO if it takes longer as in #806 to change from `Initial` -> `Ready` with a running game controller. 
And can use `Control.additional_outputs.referee_pose_queue` in twix to see the queue.
